lib/components/split-pane.js

```diff
@@ -134,80 +133,79 @@ export default class SplitPane extends PureComponent {
                 onMouseDown={this.handleDragStart}
                 onDoubleClick={this.handleAutoResize}
                 style={{backgroundColor: borderColor}}
-                className={css('divider', `divider_${direction}`)}
+                className={`splitpane_divider splitpane_divider_${direction}`}
               />
             ) : null
           ];
         })}
-        <div style={{display: this.state.dragging ? 'block' : 'none'}} className={css('shim')} />
+        <div style={{display: this.state.dragging ? 'block' : 'none'}} className="splitpane_shim" />
+
+        <style jsx>{`
+          .splitpane_panes {
+            display: flex;
+            flex: 1;
+            outline: none;
+            position: relative;
+            width: 100%;
+            height: 100%;
+          }
+
+          .splitpane_panes_vertical {
+            flex-direction: row;
+          }
+
+          .splitpane_panes_horizontal {
+            flex-direction: column;
+          }
+
+          .splitpane_pane {
+            flex: 1;
+            outline: none;
+            position: relative;
+          }
+
+          .splitpane_divider {
+            box-sizing: border-box;
+            z-index: 1;
+            background-clip: padding-box;
+            flex-shrink: 0;
+          }
+
+          .splitpane_divider_vertical {
+            border-left: 5px solid rgba(255, 255, 255, 0);
+            border-right: 5px solid rgba(255, 255, 255, 0);
+            width: 11px;
+            margin: 0 -5px;
+            cursor: col-resize;
+          }
+
+          .splitpane_divider_horizontal {
+            height: 11px;
+            margin: -5px 0;
+            border-top: 5px solid rgba(255, 255, 255, 0);
+            border-bottom: 5px solid rgba(255, 255, 255, 0);
+            cursor: row-resize;
+            width: 100%;
+          }
+
+          {/*
```

Comment:
Just use CSS comments `{/* */}` is invalid css and might break very easily

Reply:
need to fix all over the PR

Reply:
I'll fix this 👌